fetchai/ledger/api/__init__.py

```diff
@@ -55,6 +59,10 @@ def __init__(self, host=None, port=None, network=None):
 
         # Check that ledger version is compatible with API version
         server_version = self.server.version().lstrip('v')
+        if server_version.startswith('Unknown version with hash'):
+            logging.warn('Using development version')
+            return
```

Comment:
`elif` may be more appropriate than return, to avoid shorting out any other logic we may add in future.

Reply:
Done.